src/Symfony/Component/Filesystem/Path.php

```diff
@@ -368,7 +368,7 @@ public static function isAbsolute(string $path): bool
         }
 
         // Strip scheme
-        if (false !== $schemeSeparatorPosition = strpos($path, '://')) {
+        if ((false !== $schemeSeparatorPosition = strpos($path, '://')) && !preg_match('#^[A-Z]://#', $path)) {
```

Comment:
let's be more lenient?
```suggestion
        if (false !== $schemeSeparatorPosition = strpos($path, '://') && 1 !== $schemeSeparatorPosition)) {
```

Reply:
works for me as well